server/testutil/testenv/testenv.go

```diff
@@ -35,9 +35,9 @@ var (
 func init() {
 	// N.B. We do this here to avoid a data race condition that happens when
 	// multiple tests Configure the logger simultaneously.
-	if err := log.Configure(log.Opts{Level: "debug", EnableShortFileName: true}); err != nil {
-		log.Fatalf("Error configuring logging: %s", err)
-	}
+	*log.LogLevel = "debug"
+	*log.IncludeShortFileName = true
+	log.Configure()
```

Comment:
I understand you're just refactoring this code, so not a comment on your changes per-se, but:

How does this init() interplay with the one in log.go? They seem to be both doing the same thing?

Maybe instead of using init() here, we could have a sync.once in testEnv that configures the logger using some method like ConfigureForTesting(&opts{debug, shortFile})?

Reply:
can you make this a sync.Once? otherwise CL LGTM